projects/plugins/jetpack/extensions/blocks/button/button.php

```diff
@@ -39,10 +39,15 @@ function register_block() {
 function render_block( $attributes, $content ) {
 	$save_in_post_content = get_attribute( $attributes, 'saveInPostContent' );
 
-	if ( Blocks::is_amp_request() ) {
```

Comment:
Note: this AMP is removed here. Previously we only loaded the block's style if it's an AMP request because the only styles we needed to load were targeted for AMP. We now have a style rule that should be applied when not in an AMP request, so I think the conditional is no longer needed.